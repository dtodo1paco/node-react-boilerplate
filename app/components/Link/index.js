/**
 * An internal link to a certain page, an anchor tag
 */
import { Link } from 'react-router-dom';
import styled from 'styled-components';

const link = styled(Link)`
  color: ${props => props.theme.palette.primary.main};
  text-transform: uppercase;
  &:hover {
    color: ${props => props.theme.palette.primary.light};
  }
`;

link.displayName = 'Link';
export default link;

// ${props => props.theme.palette.primary.main};
