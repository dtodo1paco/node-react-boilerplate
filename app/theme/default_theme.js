import grey from '@material-ui/core/colors/grey';
import red from '@material-ui/core/colors/red';
import blueGrey from '@material-ui/core/colors/blueGrey';
import lightGreen from '@material-ui/core/colors/lightGreen';

export const colors = {
  background: {
    paper: '#212121',
    default: '#202020',
    component: {
      primary: grey[300],
      secondary: blueGrey[900],
    },
  },
  foreground: {
    text: '#e0e0e0',
  },
  error: red,
  primaryColor: {
    main: '#CDDC39',
    light: lightGreen[300],
    dark: lightGreen[500],
  },
  secondaryColor: {
    main: '#b0bec5',
    light: blueGrey[300],
    dark: blueGrey[700],
  },
};
colors.textColor = {
  primary: colors.secondaryColor.light,
  secondary: colors.primaryColor.light,
  disabled: grey[200],
  hint: grey[800],
};

export default colors;
