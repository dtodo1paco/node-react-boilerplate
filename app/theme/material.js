import grey from '@material-ui/core/colors/grey';
import red from '@material-ui/core/colors/red';
import blueGrey from '@material-ui/core/colors/blueGrey';
import teal from '@material-ui/core/colors/teal';

export const colors = {
  background: {
    paper: blueGrey[300],
    default: blueGrey[100],
    component: {
      primary: blueGrey[300],
      secondary: blueGrey[100],
    },
  },
  foreground: {
    text: teal[400],
  },
  error: red,
  primaryColor: {
    main: teal[700],
    light: teal[500],
    dark: teal[900],
  },
  secondaryColor: {
    main: blueGrey[700],
    light: blueGrey[500],
    dark: blueGrey[900],
  },
};
colors.textColor = {
  primary: colors.secondaryColor.dark,
  secondary: colors.primaryColor.dark,
  disabled: grey[200],
  hint: grey[800],
};

export default colors;
