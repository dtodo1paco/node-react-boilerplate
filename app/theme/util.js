import { createMuiTheme } from '@material-ui/core/styles';
import createPalette from '@material-ui/core/styles/createPalette';

const theme = createMuiTheme();
export const createTheme = (id, type, colors) =>
  createMuiTheme({
    props: {
      id,
    },
    palette: createPalette({
      type,
      background: colors.background,
      text: colors.textColor,
      primary: colors.primaryColor,
      secondary: colors.secondaryColor,
      error: colors.error,
      // Used by `getContrastText()` to maximize the contrast between the background and
      // the text.
      contrastThreshold: 3,
      // Used to shift a color's luminance by approximately
      // two indexes within its tonal palette.
      // E.g., shift from Red 500 to Red 300 or Red 700.
      tonalOffset: 0.2,
      status: {
        danger: colors.primaryColor,
      },
    }),
    overrides: {
      MuiAppBar: {
        colorPrimary: {
          backgroundColor: colors.background.component.secondary,
        },
      },
      MuiButton: {
        textSecondary: {
          '&:hover': {
            backgroundColor: colors.secondaryColor.dark,
          },
        },
        outlined: {
          backgroundColor: colors.background.paper,
          color: colors.primaryColor.dark,
          '&:hover': {
            backgroundColor: colors.primaryColor.dark,
            color: colors.background.paper,
          },
        },
      },
      MuiButtonBase: {
        root: {
          verticalAlign: 'text-top',
        },
      },
      MuiFormControl: {
        root: {
          margin: theme.spacing.unit,
          minWidth: 120,
          maxWidth: 240,
        },
      },
      MuiFormGroup: {
        root: {
          backgroundColor: colors.background.component.secondary,
          color: colors.primaryColor,
          padding: '2rem',
        },
      },
      MuiFormLabel: {
        root: {
          color: colors.secondaryColor.dark,
        },
      },
      MuiFormHelperText: {
        root: {
          [theme.breakpoints.down('sm')]: {
            fontSize: 'x-small',
          },
        },
      },
      MuiInput: {
        input: {
          color: colors.primaryColor.light,
        },
      },
      MuiList: {
        root: {
          margin: 'inherit',
          '&:focus': {
            outlineColor: colors.secondaryColor.dark,
          },
        },
        padding: {
          paddingBottom: 0,
        },
      },
      MuiListItemIcon: {
        root: {
          color: colors.primaryColor.main,
        },
      },
      MuiListItemText: {
        root: {
          padding: 0,
        },
      },
      MuiPaper: {
        root: {
          backgroundColor: colors.background.component.secondary,
          '&:focus': {
            outlineColor: 'inherit',
          },
        },
      },
      MuiPopover: {
        paper: {
          color: colors.primaryColor.light,
        },
      },
      MuiSelect: {
        select: {
          color: colors.secondaryColor.light,
        },
      },
      MuiSnackbarContent: {
        message: {
          maxWidth: '85%',
        },
      },
      MuiSvgIcon: {
        root: {
          color: colors.primaryColor.main,
        },
      },
      MuiTableCell: {
        root: {
          [theme.breakpoints.down('sm')]: {
            padding: theme.spacing.unit,
          },
        },
      },
      MuiToolbar: {
        regular: {
          minHeight: 'unset',
          [theme.breakpoints.down('xl')]: {
            minHeight: 'unset',
          },
        },
      },
      MuiTypography: {
        headline: {
          fontSize: 24,
          color: colors.primaryColor.main,
          outline: 'none',
        },
        title: {
          fontSize: 20,
          color: colors.primaryColor.main,
          outline: 'none',
        },
        subheading: {
          fontSize: 12,
          outline: 'none',
        },
        display1: {
          fontSize: 24,
          textAlign: 'center',
          marginBottom: 8,
          outline: 'none',
        },
        display2: {
          fontSize: 20,
          whiteSpace: 'pre-wrap',
          textAlign: 'center',
          marginBottom: 8,
          outline: 'none',
          '& @media (max-width: 600px)': {
            fontSize: theme.typography.pxToRem(12),
          },
        },
        display3: {
          fontSize: 16,
          outline: 'none',
          textAlign: 'center',
          marginBottom: 8,
        },
        display4: {
          fontSize: 8,
          outline: 'none',
          margin: `${theme.spacing.unit}px auto`,
          '&:focus': {
            outlineColor: 'inherit',
          },
        },
      },
    },
  });
