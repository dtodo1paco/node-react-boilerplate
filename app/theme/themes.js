import { createTheme } from './util';
/* COLORS AVAILABLE */
import DEFAULT_COLORS from './default_theme';
import MATERIAL_COLORS from './material';

export const DEFAULT_THEME = createTheme('default', 'dark', DEFAULT_COLORS);
export const MATERIAL_THEME = createTheme('material', 'light', MATERIAL_COLORS);
export const THEMES = {
  default: DEFAULT_THEME,
  material: MATERIAL_THEME,
};

export const findThemeOrDefault = themeId =>
  THEMES[themeId] ? THEMES[themeId] : DEFAULT_THEME;
