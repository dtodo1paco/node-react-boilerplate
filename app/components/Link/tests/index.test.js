/**
 * Testing our link component
 */

import React from 'react';
import TestHelper from 'utils/testHelper';
import { DEFAULT_THEME } from 'theme/themes';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Link from '../index';

const href = 'https://dtodo1paco.org/';
const defaultProps = {
  href: '/',
  theme: DEFAULT_THEME,
  to: href,
};

const children = <h1>Test</h1>;
const getComponent = props => <Link {...props}>{children}</Link>;
const buildComponent = props =>
  props ? getComponent(props) : getComponent(defaultProps);
const buildWithRouter = props => (
  <MemoryRouter>{buildComponent(props)}</MemoryRouter>
);
const renderComponent = props =>
  TestHelper.shallow(buildWithRouter(props)).dive();
const findComponent = props => renderComponent(props).find(Link);

describe('<Link />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer
      .create(buildWithRouter(defaultProps))
      .toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
  it('should render a <Link> as base element', () => {
    const renderedComponent = findComponent(defaultProps);
    expect(renderedComponent).toHaveLength(1);
  });

  it('should have a href attribute', () => {
    const renderedComponent = findComponent(defaultProps);
    expect(renderedComponent.prop('href')).toBe(defaultProps.href);
  });

  it('should adopt a valid attribute (target)', () => {
    const props = defaultProps;
    props.target = '_blank';
    const renderedComponent = findComponent(props);
    expect(renderedComponent.prop('target')).toBe(props.target);
  });

  it('should adopt a valid attribute (type)', () => {
    const props = defaultProps;
    props.type = 'text/html';
    const renderedComponent = findComponent(props);
    expect(renderedComponent.prop('type')).toBe(props.type);
  });

  it('should have the correct children', () => {
    const renderedComponent = findComponent(defaultProps);
    expect(renderedComponent.children().type()).toBe('h1');
  });
});
