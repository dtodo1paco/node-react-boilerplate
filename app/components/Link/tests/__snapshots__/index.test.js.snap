// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should match the snapshot 1`] = `
<a
  className="Link__link-jPhLRD dbzdtS"
  href="/https://dtodo1paco.org/"
  onClick={[Function]}
  theme={
    Object {
      "breakpoints": Object {
        "between": [Function],
        "down": [Function],
        "keys": Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ],
        "only": [Function],
        "up": [Function],
        "values": Object {
          "lg": 1280,
          "md": 960,
          "sm": 600,
          "xl": 1920,
          "xs": 0,
        },
        "width": [Function],
      },
      "direction": "ltr",
      "mixins": Object {
        "gutters": [Function],
        "toolbar": Object {
          "@media (min-width:0px) and (orientation: landscape)": Object {
            "minHeight": 48,
          },
          "@media (min-width:600px)": Object {
            "minHeight": 64,
          },
          "minHeight": 56,
        },
      },
      "overrides": Object {
        "MuiAppBar": Object {
          "colorPrimary": Object {
            "backgroundColor": "#263238",
          },
        },
        "MuiButton": Object {
          "outlined": Object {
            "&:hover": Object {
              "backgroundColor": "#8bc34a",
              "color": "#212121",
            },
            "backgroundColor": "#212121",
            "color": "#8bc34a",
          },
          "textSecondary": Object {
            "&:hover": Object {
              "backgroundColor": "#455a64",
            },
          },
        },
        "MuiButtonBase": Object {
          "root": Object {
            "verticalAlign": "text-top",
          },
        },
        "MuiFormControl": Object {
          "root": Object {
            "margin": 8,
            "maxWidth": 240,
            "minWidth": 120,
          },
        },
        "MuiFormGroup": Object {
          "root": Object {
            "backgroundColor": "#263238",
            "color": Object {
              "contrastText": "rgba(0, 0, 0, 0.87)",
              "dark": "#8bc34a",
              "light": "#aed581",
              "main": "#CDDC39",
            },
            "padding": "2rem",
          },
        },
        "MuiFormHelperText": Object {
          "root": Object {
            "@media (max-width:959.95px)": Object {
              "fontSize": "x-small",
            },
          },
        },
        "MuiFormLabel": Object {
          "root": Object {
            "color": "#455a64",
          },
        },
        "MuiInput": Object {
          "input": Object {
            "color": "#aed581",
          },
        },
        "MuiList": Object {
          "padding": Object {
            "paddingBottom": 0,
          },
          "root": Object {
            "&:focus": Object {
              "outlineColor": "#455a64",
            },
            "margin": "inherit",
          },
        },
        "MuiListItemIcon": Object {
          "root": Object {
            "color": "#CDDC39",
          },
        },
        "MuiListItemText": Object {
          "root": Object {
            "padding": 0,
          },
        },
        "MuiPaper": Object {
          "root": Object {
            "&:focus": Object {
              "outlineColor": "inherit",
            },
            "backgroundColor": "#263238",
          },
        },
        "MuiPopover": Object {
          "paper": Object {
            "color": "#aed581",
          },
        },
        "MuiSelect": Object {
          "select": Object {
            "color": "#90a4ae",
          },
        },
        "MuiSnackbarContent": Object {
          "message": Object {
            "maxWidth": "85%",
          },
        },
        "MuiSvgIcon": Object {
          "root": Object {
            "color": "#CDDC39",
          },
        },
        "MuiTableCell": Object {
          "root": Object {
            "@media (max-width:959.95px)": Object {
              "padding": 8,
            },
          },
        },
        "MuiToolbar": Object {
          "regular": Object {
            "@media (min-width:0px)": Object {
              "minHeight": "unset",
            },
            "minHeight": "unset",
          },
        },
        "MuiTypography": Object {
          "display1": Object {
            "fontSize": 24,
            "marginBottom": 8,
            "outline": "none",
            "textAlign": "center",
          },
          "display2": Object {
            "& @media (max-width: 600px)": Object {
              "fontSize": "0.75rem",
            },
            "fontSize": 20,
            "marginBottom": 8,
            "outline": "none",
            "textAlign": "center",
            "whiteSpace": "pre-wrap",
          },
          "display3": Object {
            "fontSize": 16,
            "marginBottom": 8,
            "outline": "none",
            "textAlign": "center",
          },
          "display4": Object {
            "&:focus": Object {
              "outlineColor": "inherit",
            },
            "fontSize": 8,
            "margin": "8px auto",
            "outline": "none",
          },
          "headline": Object {
            "color": "#CDDC39",
            "fontSize": 24,
            "outline": "none",
          },
          "subheading": Object {
            "fontSize": 12,
            "outline": "none",
          },
          "title": Object {
            "color": "#CDDC39",
            "fontSize": 20,
            "outline": "none",
          },
        },
      },
      "palette": Object {
        "action": Object {
          "active": "#fff",
          "disabled": "rgba(255, 255, 255, 0.3)",
          "disabledBackground": "rgba(255, 255, 255, 0.12)",
          "hover": "rgba(255, 255, 255, 0.1)",
          "hoverOpacity": 0.1,
          "selected": "rgba(255, 255, 255, 0.2)",
        },
        "augmentColor": [Function],
        "background": Object {
          "component": Object {
            "primary": "#e0e0e0",
            "secondary": "#263238",
          },
          "default": "#202020",
          "paper": "#212121",
        },
        "common": Object {
          "black": "#000",
          "white": "#fff",
        },
        "contrastThreshold": 3,
        "divider": "rgba(255, 255, 255, 0.12)",
        "error": Object {
          "100": "#ffcdd2",
          "200": "#ef9a9a",
          "300": "#e57373",
          "400": "#ef5350",
          "50": "#ffebee",
          "500": "#f44336",
          "600": "#e53935",
          "700": "#d32f2f",
          "800": "#c62828",
          "900": "#b71c1c",
          "A100": "#ff8a80",
          "A200": "#ff5252",
          "A400": "#ff1744",
          "A700": "#d50000",
          "contrastText": "#fff",
          "dark": "#d32f2f",
          "light": "#e57373",
          "main": "#f44336",
        },
        "getContrastText": [Function],
        "grey": Object {
          "100": "#f5f5f5",
          "200": "#eeeeee",
          "300": "#e0e0e0",
          "400": "#bdbdbd",
          "50": "#fafafa",
          "500": "#9e9e9e",
          "600": "#757575",
          "700": "#616161",
          "800": "#424242",
          "900": "#212121",
          "A100": "#d5d5d5",
          "A200": "#aaaaaa",
          "A400": "#303030",
          "A700": "#616161",
        },
        "primary": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "#8bc34a",
          "light": "#aed581",
          "main": "#CDDC39",
        },
        "secondary": Object {
          "contrastText": "rgba(0, 0, 0, 0.87)",
          "dark": "#455a64",
          "light": "#90a4ae",
          "main": "#b0bec5",
        },
        "status": Object {
          "danger": Object {
            "contrastText": "rgba(0, 0, 0, 0.87)",
            "dark": "#8bc34a",
            "light": "#aed581",
            "main": "#CDDC39",
          },
        },
        "text": Object {
          "disabled": "#eeeeee",
          "hint": "#424242",
          "icon": "rgba(255, 255, 255, 0.5)",
          "primary": "#90a4ae",
          "secondary": "#aed581",
        },
        "tonalOffset": 0.2,
        "type": "dark",
      },
      "props": Object {
        "id": "default",
      },
      "shadows": Array [
        "none",
        "0px 1px 3px 0px rgba(0, 0, 0, 0.2),0px 1px 1px 0px rgba(0, 0, 0, 0.14),0px 2px 1px -1px rgba(0, 0, 0, 0.12)",
        "0px 1px 5px 0px rgba(0, 0, 0, 0.2),0px 2px 2px 0px rgba(0, 0, 0, 0.14),0px 3px 1px -2px rgba(0, 0, 0, 0.12)",
        "0px 1px 8px 0px rgba(0, 0, 0, 0.2),0px 3px 4px 0px rgba(0, 0, 0, 0.14),0px 3px 3px -2px rgba(0, 0, 0, 0.12)",
        "0px 2px 4px -1px rgba(0, 0, 0, 0.2),0px 4px 5px 0px rgba(0, 0, 0, 0.14),0px 1px 10px 0px rgba(0, 0, 0, 0.12)",
        "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 5px 8px 0px rgba(0, 0, 0, 0.14),0px 1px 14px 0px rgba(0, 0, 0, 0.12)",
        "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.14),0px 1px 18px 0px rgba(0, 0, 0, 0.12)",
        "0px 4px 5px -2px rgba(0, 0, 0, 0.2),0px 7px 10px 1px rgba(0, 0, 0, 0.14),0px 2px 16px 1px rgba(0, 0, 0, 0.12)",
        "0px 5px 5px -3px rgba(0, 0, 0, 0.2),0px 8px 10px 1px rgba(0, 0, 0, 0.14),0px 3px 14px 2px rgba(0, 0, 0, 0.12)",
        "0px 5px 6px -3px rgba(0, 0, 0, 0.2),0px 9px 12px 1px rgba(0, 0, 0, 0.14),0px 3px 16px 2px rgba(0, 0, 0, 0.12)",
        "0px 6px 6px -3px rgba(0, 0, 0, 0.2),0px 10px 14px 1px rgba(0, 0, 0, 0.14),0px 4px 18px 3px rgba(0, 0, 0, 0.12)",
        "0px 6px 7px -4px rgba(0, 0, 0, 0.2),0px 11px 15px 1px rgba(0, 0, 0, 0.14),0px 4px 20px 3px rgba(0, 0, 0, 0.12)",
        "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 12px 17px 2px rgba(0, 0, 0, 0.14),0px 5px 22px 4px rgba(0, 0, 0, 0.12)",
        "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 13px 19px 2px rgba(0, 0, 0, 0.14),0px 5px 24px 4px rgba(0, 0, 0, 0.12)",
        "0px 7px 9px -4px rgba(0, 0, 0, 0.2),0px 14px 21px 2px rgba(0, 0, 0, 0.14),0px 5px 26px 4px rgba(0, 0, 0, 0.12)",
        "0px 8px 9px -5px rgba(0, 0, 0, 0.2),0px 15px 22px 2px rgba(0, 0, 0, 0.14),0px 6px 28px 5px rgba(0, 0, 0, 0.12)",
        "0px 8px 10px -5px rgba(0, 0, 0, 0.2),0px 16px 24px 2px rgba(0, 0, 0, 0.14),0px 6px 30px 5px rgba(0, 0, 0, 0.12)",
        "0px 8px 11px -5px rgba(0, 0, 0, 0.2),0px 17px 26px 2px rgba(0, 0, 0, 0.14),0px 6px 32px 5px rgba(0, 0, 0, 0.12)",
        "0px 9px 11px -5px rgba(0, 0, 0, 0.2),0px 18px 28px 2px rgba(0, 0, 0, 0.14),0px 7px 34px 6px rgba(0, 0, 0, 0.12)",
        "0px 9px 12px -6px rgba(0, 0, 0, 0.2),0px 19px 29px 2px rgba(0, 0, 0, 0.14),0px 7px 36px 6px rgba(0, 0, 0, 0.12)",
        "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 20px 31px 3px rgba(0, 0, 0, 0.14),0px 8px 38px 7px rgba(0, 0, 0, 0.12)",
        "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 21px 33px 3px rgba(0, 0, 0, 0.14),0px 8px 40px 7px rgba(0, 0, 0, 0.12)",
        "0px 10px 14px -6px rgba(0, 0, 0, 0.2),0px 22px 35px 3px rgba(0, 0, 0, 0.14),0px 8px 42px 7px rgba(0, 0, 0, 0.12)",
        "0px 11px 14px -7px rgba(0, 0, 0, 0.2),0px 23px 36px 3px rgba(0, 0, 0, 0.14),0px 9px 44px 8px rgba(0, 0, 0, 0.12)",
        "0px 11px 15px -7px rgba(0, 0, 0, 0.2),0px 24px 38px 3px rgba(0, 0, 0, 0.14),0px 9px 46px 8px rgba(0, 0, 0, 0.12)",
      ],
      "shape": Object {
        "borderRadius": 4,
      },
      "spacing": Object {
        "unit": 8,
      },
      "transitions": Object {
        "create": [Function],
        "duration": Object {
          "complex": 375,
          "enteringScreen": 225,
          "leavingScreen": 195,
          "short": 250,
          "shorter": 200,
          "shortest": 150,
          "standard": 300,
        },
        "easing": Object {
          "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
          "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
          "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
        },
        "getAutoHeightDuration": [Function],
      },
      "typography": Object {
        "body1": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 400,
          "lineHeight": "1.46429em",
        },
        "body2": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 500,
          "lineHeight": "1.71429em",
        },
        "button": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.875rem",
          "fontWeight": 500,
          "textTransform": "uppercase",
        },
        "caption": Object {
          "color": "#aed581",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "0.75rem",
          "fontWeight": 400,
          "lineHeight": "1.375em",
        },
        "display1": Object {
          "color": "#aed581",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "2.125rem",
          "fontWeight": 400,
          "lineHeight": "1.20588em",
        },
        "display2": Object {
          "color": "#aed581",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "2.8125rem",
          "fontWeight": 400,
          "lineHeight": "1.13333em",
          "marginLeft": "-.02em",
        },
        "display3": Object {
          "color": "#aed581",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "3.5rem",
          "fontWeight": 400,
          "letterSpacing": "-.02em",
          "lineHeight": "1.30357em",
          "marginLeft": "-.02em",
        },
        "display4": Object {
          "color": "#aed581",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "7rem",
          "fontWeight": 300,
          "letterSpacing": "-.04em",
          "lineHeight": "1.14286em",
          "marginLeft": "-.04em",
        },
        "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
        "fontSize": 14,
        "fontWeightLight": 300,
        "fontWeightMedium": 500,
        "fontWeightRegular": 400,
        "headline": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1.5rem",
          "fontWeight": 400,
          "lineHeight": "1.35417em",
        },
        "pxToRem": [Function],
        "round": [Function],
        "subheading": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1rem",
          "fontWeight": 400,
          "lineHeight": "1.5em",
        },
        "title": Object {
          "color": "#90a4ae",
          "fontFamily": "\\"Roboto\\", \\"Helvetica\\", \\"Arial\\", sans-serif",
          "fontSize": "1.3125rem",
          "fontWeight": 500,
          "lineHeight": "1.16667em",
        },
      },
      "zIndex": Object {
        "appBar": 1100,
        "drawer": 1200,
        "mobileStepper": 1000,
        "modal": 1300,
        "snackbar": 1400,
        "tooltip": 1500,
      },
    }
  }
>
  <h1>
    Test
  </h1>
</a>
`;
